google-cloud-pubsub/lib/google/cloud/pubsub/subscriber.rb

```diff
@@ -169,6 +170,87 @@ def stopped?
           synchronize { @stopped }
         end
 
+        ##
+        # Register to be notified of errors when raised.
```

Comment:
You might want to make it explicit that there can be at most one error handler (i.e. that calling this when a different handler is registered, will replace the old handler).

Reply:
Would you rather users be able to register multiple handlers? The problem with that is do we also need to provide a way to remove a handler? I don't want this to become a big API, which is why I made it just one. What do you think?

Reply:
Speech has the ability to register multiple `on_error` handlers, but not the ability to remove handlers.

Reply:
I added the ability to make repeated calls to `on_error` and made a note in the documentation.